<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ashutoshs-MacBook-Air.local" ignored="0" name="Login test" tests="2" failures="1" timestamp="13 Dec 2021 16:46:33 GMT" time="181.712" errors="0">
  <testcase name="createEmployee" time="92.285" classname="com.tdd.orangehrm.testcase.PIMPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [First Name Field Label is not visible] but found [true],
	expected [EmployeeID Label is not visible] but found [true],
	expected [Photograph Label is not visible] but found [true],
	expected [Create Login Detail Label is not visible] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [First Name Field Label is not visible] but found [true],
expected [EmployeeID Label is not visible] but found [true],
expected [Photograph Label is not visible] but found [true],
expected [Create Login Detail Label is not visible] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.tdd.orangehrm.testcase.PIMPageTest.createEmployee(PIMPageTest.java:90)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createEmployee -->
  <testcase name="navigateToAddEmployee" time="39.828" classname="com.tdd.orangehrm.testcase.PIMPageTest"/>
</testsuite> <!-- Login test -->
